src/integrationtests/java/com/aws/greengrass/integrationtests/ShadowManagerTest.java

```diff
@@ -169,6 +170,7 @@ void GIVEN_existing_sync_information_WHEN_config_updates_THEN_removed_sync_infor
                 .configFile(DEFAULT_CONFIG)
                 .mqttConnected(false)
                 .build());
+        shadowManager.startSyncingShadows(ShadowManager.StartSyncInfo.builder().build());
```

Comment:
This turns off the syncing that happens when the component starts up. 
Otherwise, the shadows that are not present in the config are removed (shadow-0 and shadow-1) when the syncing happens resulting in false assertions.